backend/danswer/db/pg_file_store.py

```diff
@@ -73,6 +92,22 @@ def delete_lobj_by_id(
     pg_conn.lobject(lobj_oid).unlink()
 
 
+def delete_lobj_by_name(
+    lobj_name: str,
+    db_session: Session,
+) -> None:
+    pgfilestore = db_session.query(PGFileStore).filter_by(file_name=lobj_name).first()
```

Comment:
nit: can we use `get_pgfilestore_by_file_name` here?

Reply:
no